targets/targets.json

```diff
@@ -4518,5 +4518,17 @@
         "detect_code": ["7016"],
         "release_versions": ["5"],
         "bootloader_supported": true
+    },
+    "TMPM3HQ": {
+        "inherits": ["Target"],
+        "core": "Cortex-M3",
+        "is_disk_virtual": true,
+        "extra_labels": ["TOSHIBA"],
+        "macros": ["__TMPM3HQ__"],
+        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
```

Comment:
@ganesh-ramachandran : If we are removing IAR support until mbed OS supports IAR 8.x, then "IAR" from supported_toolchains should be removed too. Thanks.

Reply:
> @ganesh-ramachandran : If we are removing IAR support until mbed OS supports IAR 8.x, then "IAR" from supported_toolchains should be removed too. Thanks.

It should compile it (needs core), so can stay as it is, just definition removal